episodes/introduction.md

```diff
@@ -1,102 +1,95 @@
 ---
-title: "Using Markdown"
-teaching: 10
-exercises: 2
+title: "Welcome"
+teaching: 30
+exercises: 30
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- What can I expect from this Training?
+- What is expected of me in this Training?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+By the end of this session, participants should be able to...
+
+- describe the format of the Trainer Training program and sessions.
+- summarise the lesson development experience of the participants in the cohort.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Introduction
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown](https://pandoc.org/MANUAL.txt) for static files and
-[R Markdown][r-markdown] for dynamic files that can render code into output. 
-Please refer to the [Introduction to The Carpentries 
-Workbench](https://carpentries.github.io/sandpaper-docs/) for full documentation.
+## Reading List
 
-What you need to know is that there are three sections required for a valid
-Carpentries lesson:
+- [The Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
 
- 1. `questions` are displayed at the beginning of the episode to prime the
-    learner for the content.
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
+## Introductions
```

Comment:
Might it be better to move intros here or will this episode not be a discussion itself?

Reply:
resolved in cf2666a